lib/googlesearch.py

```diff
@@ -23,6 +26,7 @@ def closeBrowser():
         browser.quit()
 
 def search(req, stop):
+    time.sleep(10)
```

Comment:
Waiting 10 sec at each request is ridiculous, it shouldn't be handled that way. A rotating proxy must be used to avoid getting blocked.

Reply:
Implemented the rotating proxy. Have a look